metabase/metabase
# Truncate tooltip content for table items with large descriptions

Closes https://github.com/metabase/metabase/issues/36794
Closes https://github.com/metabase/metabase/issues/36793

### Description

Truncates tooltip content for items with large descriptions when previewing in a table.

Additionally this PR updates our `Markdown` component to accept a `lineClamp` prop so that we can more easily use this functionality elsewhere.

### How to verify

1. Create a question
2. Add to the description a large amount of text
3. See in the collection view where the tooltip expands to

### Demo

<img width="531" alt="Screenshot 2024-01-09 at 11 27 22 AM" src="https://github.com/metabase/metabase/assets/7104357/8343dcde-a6fa-415f-adec-5cadad0113d6">

### Checklist
